src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -137,6 +137,16 @@
   @Fluent
   WebSocketBase writeTextMessage(String text);
 
+  /**
+   * Writes a (potentially large) piece of binary text data to the connection.
+   * This data might be written as multiple frames if it exceeds the maximum WebSocket frame size.
+   *
+   * @param textBuffer  the data to write
+   * @return a reference to this, so the API can be used fluently
+   */
+  @Fluent
+  WebSocketBase writeBinaryTextMessage(Buffer textBuffer);
```

Comment:
it's not clear whether it writes binary or test message, can you clarify ? it should either be called writeTextMessage or writeBinaryMessage

Reply:
updated